programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1543,7 +1543,7 @@ static struct option run_options[] = {
         { "dns-upstream", required_argument, NULL, 'u'},
 };
 
-static const char* ip_range = "100.64.0.0/10";
+static const char* ip_range = "100.64.0.1/10";
```

Comment:
Are you sure this works with `ip route add`? I was about to file a separate issue with regards to the present behavior building the main branch. We save `100.64.0.1` to the `/var/lib/ziti/config.json` file into `TunIpv4`. Current main branch build runs successfully the first time, when ip_range is `100.64.0.0/10`, but fails when it reads this value from the config the second time you run it (using TunIpv4 = ip_range = `100.64.0.1/10`. ) with error: 
`Error: Invalid prefix for given prefix length`
![image](https://user-images.githubusercontent.com/22971615/166612149-0c180863-4556-4c9d-aff5-ed2fd5aa3701.png)

+cc @dovholuknf

Reply:
I've only tested on Windows so far. Thanks for the catch.

I would thin k you see the same problem if you set the dns range to an address with non-zero host bits via `-d`?

I think the linux netif driver needs to make sure that it uses only the network portion of the address when adding the intercept route. I'll update accordingly.

Reply:
I just tested this. So it appears we ignore `-d` once the `config.json` file exists. 
![image](https://user-images.githubusercontent.com/22971615/166613392-0f6acbaf-0d6f-48d5-852d-0d445d7abec2.png)

Trying again after removing the file, it does use the `-d`  flag (until we run it again), and does choke on the `ip_range` as well.
![image](https://user-images.githubusercontent.com/22971615/166613482-4079b5a3-e4e7-4efb-b8b8-963ff2729961.png)

Reply:
thanks again. IMO we should prioritize the command line options over values in a config file (especially when it is loaded implicitly). I'm not sure if that change will fit in this PR but I'll think about it.

Reply:
@sabedevops you'll no longer get the ip route error when using -d (in the absence of a config file)

Reply:
In case others needed to look it up too, `-d` is the abbreviated form of `--dns-ip-range`.